src/helpers/helpers.core.js

```diff
@@ -27,3 +27,19 @@ export function getElementCenterPoint(element, useFinalPosition) {
   const {x, y} = element.getProps(['x', 'y'], useFinalPosition);
   return {x, y};
 }
+
+const isOlderPart = (act, req) => req > act || (act.length > req.length && act.substr(0, req.length) === req);
+
+export function requireVersion(pkg, min, ver) {
+  const parts = ver.split('.');
+  let i = 0;
+  for (const req of min.split('.')) {
+    const act = parts[i++];
+    if (parseInt(req, 10) < parseInt(act, 10)) {
+      break;
+    }
+    if (isOlderPart(act, req)) {
+      throw new Error(`${pkg} v${ver} is not supported. v${min} or newer is required.`);
```

Comment:
@kurkle maybe I'm wrong but this looks like a breaking change.
We are forcing all users to use CHART.JS 3.7.0. Maybe a warning could be emitted and in the version 2.0.0 of the plugin could be an error.
Furthermore, afa I understood, this strong dependency is related to the use of sub elements, currently used only in polygons and then, if a user is not using polygons, is anyway forced to change.

Reply:
@kurkle if we want to maintain it, we should also add a note in the doc and change the README.

Reply:
You are probably right, we could do with a warning. The requirement is related to fallback from array options. So applies to use cases that proved annotations as array too. Those "work", but defaults come from line annotation and not from the actual type.

The workaround this PR removed from `afterRegister` needs to be restored and verify it contains sub-objects of every annotation types defaults.